types/three/examples/jsm/webxr/XRHandModelFactory.d.ts

```diff
@@ -1,9 +1,17 @@
 import { Group, Object3D } from '../../../src/Three';
 
+import { XRHandPrimitiveModel } from './XRHandPrimitiveModel';
+import { XRHandOculusMeshModel } from './XRHandOculusMeshModel';
+
 export class XRHandModel extends Object3D {
     constructor();
 
-    motionController: any;
+    motionController: XRHandPrimitiveModel | XRHandOculusMeshModel;
+}
+
+export interface HandModelOptions {
+    model?: 'lowpoly';
+    primitive?: 'sphere' | 'box';
 }
```

Comment:
I think this should be split two different interfaces:the options for `XRHandOculusMeshModel` and the options for `XRHandPrimitiveModel` since they don't seem to overlap at all. And then the `options` parameter for `createHandModel` should be the options for `XRHandOculusMeshModel`, since the provided options never get passed to `XRHandPrimitiveModel`.